core/src/main/scala/cats/Applicative.scala

```diff
@@ -38,6 +38,14 @@ import simulacrum.typeclass
   def replicateA[A](n: Int, fa: F[A]): F[List[A]] =
     sequence(List.fill(n)(fa))
 
+  /** Returns the given argument if `cond` is `false`, otherwise, unit lifted into F. */
+  def unlessA[A](cond: Boolean)(f: => F[A]): F[Unit] =
```

Comment:
Ok I'll change those to strict + `Eval`. Honestly I have never had a case where it mattered, but scalaz does it this way so it's probably worthwhile to have the nonstrict version.